lotusrb.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'lotus-utils',      '~> 0.3', '>= 0.3.4'
   spec.add_dependency 'lotus-router',     '~> 0.2', '>= 0.2.1'
   spec.add_dependency 'lotus-controller', '~> 0.3', '>= 0.3.2'
+  spec.add_dependency 'lotus-model',      '~> 0.2.2', '>= 0.2.2'
```

Comment:
Can you please add point lotus-model to my migration branch for the time being? It would help you do integration test. FYI, you have to specify it in Gemfile instead, gemspec does not allow pointing to github repo.